bin/sl-pm-install.txt

```diff
@@ -30,6 +30,8 @@ Options:
   --skip-default-install
                       Disable 'npm install' phase of application deployments
                       made on this PM.
+  --json-file-db      Persist deployment information and metrics in a JSON based
+                      database. Not recommended for production use.
```

Comment:
What do you think about not adding this option to `sl-pm-install`? If someone is using `sl-pm-install` they are likely trying to follow our "recommended production use", so there's no real reason to add this convenience to opt-out of that, is there?